src/Engine/Waypoint/Waypoints.cpp

```diff
@@ -395,3 +395,31 @@ Waypoints::AddTakeoffPoint(const GeoPoint& location,
 
   Optimise();
 }
+
+Waypoint
+Waypoints::GenerateGotoPoint(const GeoPoint& location,
+                                const double terrain_alt) const
+{
+  Waypoint goto_point(location);
+  goto_point.elevation = terrain_alt;
+  goto_point.name = _T("(goto)");
```

Comment:
The name can be adjusted, I was after something short.

Reply:
I used `(goto)` since it was the same pattern used by `(takeoff)`, but it has 2 downside:
- the final bracket gets cut off when using 5 letters display `(goto` (same issue as `(take`)
- the arguments for the 2 related events slightly differ, which could potentially lead to confusion. 
  It would be much nicer allowing:
  ```
  event=GotoLookup goto
  event=AddWaypoint goto
  ```